fedimint-server/src/config/api.rs

```diff
@@ -652,6 +653,15 @@ pub fn server_endpoints() -> Vec<ApiEndpoint<ConfigGenApi>> {
                 }
             }
         },
+        api_endpoint! {
```

Comment:
This full vs non-full seems unnecessary. Should we merge them?

Maybe this one could be `peer_status` and the other one `dkg_status` or `server_status?